tests/bootstrap.php

```diff
@@ -106,42 +116,38 @@ function _configure_wp_parsely_specified_version() {
 	}
 }
 
-switch ( getenv( 'WPVIP_PARSELY_INTEGRATION_PLUGIN_VERSION' ) ) {
```

Comment:
No need of this `_configure_specify_wp_parsely_version` function is doing the same.